olp-cpp-sdk-dataservice-read/tests/QuadTreeIndexTest.cpp

```diff
@@ -100,6 +100,18 @@ TEST(QuadTreeIndexTest, ParseBlob) {
     EXPECT_TRUE(data3 == boost::none);
   }
 
+  {
+    SCOPED_TRACE("Quad tree ConvertToIndexData test");
```

Comment:
```suggestion
    SCOPED_TRACE("Quad tree ConvertToIndexData test");
    
```
Newline after SCOPED_TRACE. Please keep consistent!